jujugui/templates/index.html.mako

```diff
@@ -399,30 +400,25 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
           // We need to activate the hotkeys when running the application
           // in production. Unit tests should call it manually.
           app.activateHotkeys();
-
-          window.ga_id = juju_config.GA_key || '';
-          if (window.ga_id != '') {
-            _gaq.push(['_setAccount', window.ga_id]);
-            _gaq.push (['_gat._anonymizeIp']);
-            _gaq.push(['_setDomainName', 'none']);
-            _gaq.push(['_setAllowLinker', true]);
-            _gaq.push(['_trackPageview']);
-          }
         });
 
       };
     </script>
-    <script async type="text/javascript">
-      (function() {
-      var ga = document.createElement('script');
-      ga.type = 'text/javascript'; ga.async = true;
-      ga.src = ('https:' == document.location.protocol ?
-                'https://ssl' : 'http://www') +
-                '.google-analytics.com/ga.js';
-      var s = document.getElementsByTagName('script')[0];
-      s.parentNode.insertBefore(ga, s);
-      })();
+
+
+    <!-- Google Tag Manager -->
+    <noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-K9KCMZ"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    <script>
+    if (window.GA_key || juju_config.GA_key) {
```

Comment:
Thanks for moving this to the bottom but in sandbox mode it appears that GA_key is not set so it never actually gets run.